packages/cli/README.md

```diff
@@ -31,72 +31,264 @@ npx @apillon/cli <command> [options]
+  }
+]
 ```
 
-To deploy the website to the staging environment, use the --preview flag:
+#### `storage upload`
+This command uploads files to a specified bucket.
 
-```bash
-apillon hosting deploy-website ./dist --uuid your-website-uuid --key your-api-key --secret your-api-secret --preview
-# or
-npx @apillon/cli hosting deploy-website ./dist --uuid your-website-uuid --key your-api-key --secret your-api-secret --preview
+**Options**
+- `<file-path>`: Path to the folder containing your files.
+- `-b, --bucket-uuid <string>`: UUID of the bucket to upload files to.
+
+**Example**
+```sh
+apillon storage upload --uuid your-bucket-uuid ./my_folder
 ```
+#### `storage get-file`
 
-## Environment Variables
+This command retrieves information about a specific file in a bucket.
 
-You can use environment variables to set the API URL, API key, and API secret:
+**Options**
+- `-b, --bucket-uuid <string>`: UUID of the bucket.
+- `--file-uuid <string>`: UUID or CID of the file to retrieve.
 
-- `APILLON_API_URL`: Apillon API URL.
-- `APILLON_API_KEY`: Apillon API key.
-- `APILLON_API_SECRET`: Apillon API secret.
+#### `storage delete-file`
 
-## Help
+This command deletes a specific file from a bucket.
 
-To display the help information for the CLI or a specific command, use the `-h` or `--help` option:
+**Options**
+- `-b, --bucket-uuid <string>`: UUID of the bucket.
+- `--file-uuid <string>`: UUID or CID of the file to delete.
+
+## `NFTs`
+
+#### `nfts list-collections`
+This command lists all NFT collections owned by the project related to the API key.
+
+**Options**
+- `--status <integer>`: UUID of the collection to retrieve (CollectionStatus enum, optional).
 
+#### `nfts get-collection`
+This command retrieves information about a specific NFT collection.
+
+**Options**
+- `--uuid <collection-uuid>`: UUID of the collection to retrieve.
+
+#### `nfts create-collection`
+This command creates a new NFT collection. The JSON file needs to have the property structure as type `ICreateCollection`, which can be found in the SDK docs.
+
+**Options**
+- `<file-path>`: Path to the JSON data file for the new collection.
+
+**Example**
 ```sh
-apillon -h
-apillon hosting -h
-npx @apillon/cli hosting deploy-website --help
+apillon nfts create-collection ./nft-data.json
 ```
 
+#### `nfts mint-nft`
+This command mints NFTs for a collection with a specific UUID.
+
+**Options**
+- `--uuid <collection-uuid>`: UUID of the collection to mint NFTs to.
+- `-a, --address <string>`: Address which will receive minted NFTs.
+- `-q --quantity <integer>`: Number of NFTs to mint.
+
+#### `nfts nest-mint-nft`
+This command nest mints NFT child collection to a parent collection with a specific UUID and parent NFT with id.
+
+**Options**
+- `-c, --parent-colleciton-uuid <collection-uuid>`: Parent collection UUID to which child NFTs will be minted to.
+- `-pid, --parent-nft-id <string>`: Parent collection NFT id to which child NFTs will be minted to.
```

Comment:
Shorthands can be only single letter! (see commander docs).

Alternatevly I suggest renaiming paramers for somethong like this:
```
--parent-collection <parent-collection-uuid>
--parent-nft <destination-nft-token-id>
-q, --quantity <int>
```
I believe there are missing options here like: 
```
--child-collection <child-collection-uuid>
--child-nft <child-nft-itoken-d>
```